### PR TITLE
Balloon alerts on Penthrite

### PR DESCRIPTION
**Pull request by aaaa1023**
## About The Pull Request

 Adds balloon alerts to when Penthrite begins processing and when it finishes processing and leaves your system.   Penthrite is mostly a chem that is used for its ability to negate falling into crit so very noticeable feedback that lets you know when you are no longer crit immune seems important. This is mostly for miners because it is very easy to run out of penthrite without knowing and get critted when you thought you were still crit immune.

## Why It's Good For The Game

More feedback on a chemical that miners often rely on to not die and makes it less of a guessing game of when you need to use another pen.

## Changelog
:cl:
add: Penthrite now has Balloon alerts for when it first starts processing and when it finishes processing.
/:cl:

### CODE REVIEW

**Comment by KittyNoodle** [comment]
you should do the same with nooartium

File: code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm [hunk 1]
```diff
@@ -514,13 +514,13 @@
...
 
-/datum/reagent/medicine/c2/penthrite/on_mob_metabolize(mob/living/M)
+/datum/reagent/medicine/c2/penthrite/on_mob_metabolize(mob/living/user)
 	. = ..()
-	to_chat(M,"<span class='notice'>Your heart begins to beat with great force!")
-	ADD_TRAIT(M, TRAIT_STABLEHEART, type)
-	ADD_TRAIT(M, TRAIT_NOHARDCRIT,type)
-	ADD_TRAIT(M, TRAIT_NOSOFTCRIT,type)
-	ADD_TRAIT(M, TRAIT_NOCRITDAMAGE,type)
+	user.balloon_alert(user, "Your heart begins to beat with great force!")
```
Comment (Mothblocks) [inline code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm:519]:
```suggestion
	user.balloon_alert(user, "your heart beats with a great force")
```

File: code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm [hunk 2]
```diff
@@ -549,11 +549,12 @@
 		volume = 0
 	. = ..()
 
-/datum/reagent/medicine/c2/penthrite/on_mob_end_metabolize(mob/living/M)
-	REMOVE_TRAIT(M, TRAIT_STABLEHEART, type)
-	REMOVE_TRAIT(M, TRAIT_NOHARDCRIT,type)
-	REMOVE_TRAIT(M, TRAIT_NOSOFTCRIT,type)
-	REMOVE_TRAIT(M, TRAIT_NOCRITDAMAGE,type)
+/datum/reagent/medicine/c2/penthrite/on_mob_end_metabolize(mob/living/user)
+	user.balloon_alert(user, "Your heart relaxes and begins to beat calmly.")
```
Comment (Mothblocks) [inline code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm:553]:
```suggestion
	user.balloon_alert(user, "your heart relaxes")
```

**Review by Mothblocks — CHANGES_REQUESTED** [review]
Balloon alerts should be all lowercase and concise.

**Comment by Mothblocks** [comment]
Seems like a good change though.

**Pull request merged**